cypress/support/pages/CommonPage.ts

```diff
@@ -0,0 +1,69 @@
+import Masthead from "./admin_console/Masthead";
+import SidebarPage from "./admin_console/SidebarPage";
+import TabPage from "./admin_console/components/TabPage";
+import FormPage from "./admin_console/components/FormPage";
+import ModalUtils from "../util/ModalUtils";
+import ActionToolbarPage from "./admin_console/components/ActionToolbarPage";
+import TableToolbarPage from "./admin_console/components/TableToolbarPage";
+import TablePage from "./admin_console/components/TablePage";
+import EmptyStatePage from "./admin_console/components/EmptyStatePage";
+
+export default class CommonPage {
+  private mastheadPage: Masthead;
```

Comment:
Since these are instantiated immediately you can get rid of the constructor and initialize them inline.
```suggestion
  private mastheadPage = new Masthead();
```